services/dumper.js

```diff
@@ -163,11 +163,14 @@ function Dumper(config) {
       };
     });
 
+    const modelSafeName = stringUtils.transformToSafeString(table);
+
     copyHandleBarsTemplate({
       source: `app/models/${config.dbDialect === 'mongodb' ? 'mongo' : 'sequelize'}-model.hbs`,
       target: `models/${tableToFilename(table)}.js`,
       context: {
-        modelName: stringUtils.pascalCase(table),
+        modelVariableName: stringUtils.pascalCase(modelSafeName),
```

Comment:
I don't understand what is the purpose of this new contextual variable.
From my understanding it adds more complexity (and diff in this PR) for a value that I don't get.
Can't we keep using the `modelName` in the template?

Reply:
I thinked we wanted
```js
const Model = sequelize.define('model', {
```

take this example :
the table name is `1`
the safe model name is `model1`

do you prefer (which is the current solution)
```js
const Model1 = sequelize.define('model1', {
```

or

```js
const model1 = sequelize.define('model1', {
```

Reply:
ok, yes we want:
`const Model1 = sequelize.define('model1', {`
Forget my comment. I didn't get it because of the "refactoring" included in the change:
`modelName` renamed in `modelVariableName`.